src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { bytes_to_hex, Sha256 } from "asmcrypto.js"
+import { TFileInformation, TdrFile } from "./types"
+
+export const getFileInformation: TFileInformation = async (
+  files,
```

Comment:
If this takes an array of files, presumably the client will have to do the work of recursively iterating through a directory? I wonder if this library should take care of that too?

Reply:
We don't need to iterate over the directory if we stick with an file input element, you only need to do `event.currentTarget.fileinputid.files` and it lists everything. I think the iteration was for the drag and drop which hopefully we're not using again. 
If we leave it as an array of files, it will make splitting up the requests easier if we get a large number of files, we can pass in slices of the full arary individually It. makes it a bit more generic for others to use as well in that you can pass any array of File like objects and it will work (ish)

Reply:
Ah, I'd forgotten the recursion was just in the drag-and-drop. Thanks!